Makefile

```diff
@@ -1,11 +1,15 @@
 WERROR?=-Werror
 PKGS=sdl2
-# TODO(#118): try to enable -Weverything and disable things that don't make sense
 CFLAGS=-Wall -Wextra $(WERROR) -pedantic -I.
-CXXFLAGS=$(CFLAGS) -std=c++17 -fno-exceptions -Wno-missing-braces -Wswitch-enum `pkg-config --cflags $(PKGS)`
+ifeq ($(OS),Windows_NT)
+	CXXFLAGS=$(CFLAGS) -std=c++17 -fno-exceptions -Wno-missing-braces -Wswitch-enum $(shell pkg-config --cflags $(PKGS)) -lmingw32 -lSDL2main -lSDL2
```

Comment:
Is it really need to specify `-lSDL2main` and `-lSDL2`. These kinds of flags are usually supplied by `pkg-config --libs sdl2` command.